services/director/tests/test_producer.py

```diff
@@ -95,7 +96,7 @@ async def push_start_services(number_comp, number_dyn, dependant=False):
     # teardown stop the services
     for service in started_services:
         service_uuid = service["service_uuid"]
-        await producer.stop_service(aiohttp_mock_app, service_uuid, True)
+        await producer.stop_service(aiohttp_mock_app, service_uuid, save_state=False)
```

Comment:
why did this change here?

Reply:
The test service behind does not implement `/state`.  Until now, this was not an issue because failing save-state was just dropping a message and ignored.